python/fastexcel/__init__.py

```diff
@@ -67,17 +67,26 @@ def load_sheet_by_name(
         *,
         header_row: int | None = 0,
         column_names: list[str] | None = None,
+        skip_rows: int = 0,
+        n_rows: int | None = None,
     ) -> ExcelSheet:
         """Loads a sheet by name.
 
         `header_row` is the index of the row containing the column labels, default index is 0.
         If `None`, the sheet does not have any column labels.
         `column_names` overrides headers found in the document.
         If `column_names` is used `header_row` will be ignored.
+        `n_rows` is the number of rows to parse.
+        `skip_rows` is the line numbers to skip after the headers.
+        If `header_row` is `None` then it skips the number of lines from 0.
```

Comment:
Shouldn't `skip_rows` be applied before the headers ? Or is it to support empty lines before and after the header ?

Reply:
I think the API is reasonably easy.
You can use a header row as column labels or not and you can paginate the results.
I did some example in the test : 
```python
sheet = excel_reader.load_sheet(
        0,
        n_rows=1000000,
        skip_rows=1,
        header_row=None,
        column_names=["This", "Is", "Amazing", "Stuff"],
    )
```

Reply:
I reckon it's way better after the header. You can already specify the header row.

Reply:
Yes, but I feel like we can specify too much information here: both `skip_rows` and `header_row` allow to specify a  number of rows to skip. Maybe we could replace `header` with `list[str] | Literal['read'] | None`. Meaning "That's the column names / read the header from the first row / No header". `skip_rows` would be skipped from the start of the file.

I might be wrong, but I haven't seen a case yet where we need to skip rows after the header.

Pandas implem does not seem to allow skipping rows before and atfer the header:
![image](https://user-images.githubusercontent.com/17085536/198222394-f6c8caf4-96bc-441f-ab6b-385fc066abb5.png)

![image](https://user-images.githubusercontent.com/17085536/198222538-a9116280-afc3-4dcb-a283-e00c4643cd20.png)

Reply:
Skip rows after header is useful to have pagination (like a regular offset)

Reply:
I'm brain dead this morning, didn't think about pagination at all :man_facepalming:  Forget what I said!

Reply:
We have the same behavior in fastexecel. The difference is that we need to be explicit by passing the header_row to `None`. I find it less confusing than pandas.